workstation.md

```diff
@@ -154,7 +154,160 @@ And you should see the exercism.io site running!  This is your local copy of the
+git clone git@github.com:nellshamrell/exercism.io.git
+```
+Then change directory into the code base:
+```bash
+cd exercism.io
+```
+
+## Installing Ruby
+We will be using rvm. Download rvm with these commands:
+```bash
+gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+```
+```bash
+\curl -sSL https://get.rvm.io | bash -s stable
+```
+```bash
+echo "source $HOME/.rvm/scripts/rvm" >> ~/.bash_profile
+```
+In order to use rvm on ubuntu, you need to have a login shell. In the terminal's menu at the top of the screen, select Edit->Profile Preferences. Select the Command tab, and check the box that says "Run Command as Login Shell".
+
+Now, you must restart your terminal for rvm to work. Type `exit` then hit `CTRL-ALT-T` to reopen the terminal.
+Now, typing `rvm list` should display the following
+>rvm rubies
+>
+>#No rubies installed yet. Try 'rvm help install'.
+
+For this project, we will use Ruby version 2.2.2. Install it using 
+```bash
+rvm install ruby-2.2.2
+```
+
+Type `rvm list` again and it should display
+
+>   rvm rubies
+>
+>       ruby-2.2.2
+
+Now we have rvm and the correct version of Ruby installed.
+
+We have to set it as the default version to use now. Execute the following command:
+```bash
+rvm --default use 2.2.2
+```
+
+## Installing Node
+```bash
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+sudo apt-get install -y nodejs
+```
+
+## Installing and setting up PostgreSQL
+```bash
+sudo apt-get install postgresql postgresql-contrib
+```
+The setup script also requires another related package, so let's install that now
+```bash
+sudo apt-get install libpq-dev
+```
+You need to set yourself up as a postgres user. Do so with this command:
+```bash
+sudo -u postgres createuser --superuser $USER
+```
+Run postgres with the following command
+```bash
+sudo -u postgres psql postgres
+```
+You should see the postgres prompt come up. It looks like
+>postgres=# 
+
+In the postgres prompt, ensure your user was created. Do this by typing in `\du`:
+>postgres=# `\du`
+
+This will display the table of users. It should include a user called postgres and another called the same user as your username:
+
+> Role name |                         Attributes                         | Member of 
+>-----------+------------------------------------------------------------+-----------
+>
+> username  | Superuser, Create role, Create DB                          | {}
+>
+> postgres  | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+
+Press `q` to exit the table.
+
+Next, we need to set a password for the new postgres account. It is easiest to use the same password that is specified in the exercism.io/config/database.yml because the setup script will ask you for several passwords in a row without mentioning which password is for which database user. Open the datbase.yml file with your favorite text editor (I'm partial to gedit): 
+```bash
+gedit ~/exercism.io/config/database.yml
```

Comment:
It would be good to specify that this should be in a different shell outside of the postgres prompt.